packages/core/src/service/Config.ts

```diff
@@ -41,6 +64,15 @@ export interface EnvConfig {
 	 * to a data pack folder or data pack archive (e.g. `.zip` or `.tar.gz`), or a special string like `@vanilla-mcdoc`.
 	 */
 	dependencies: string[]
+	/**
+	 * **Experimental feature, breaking changes could occur.**
+	 *
+	 * Custom resources, currently only works for `minecraft:resource` JSON definitions dispatched from mcdoc.
+	 */
+	customResources: [
+		filePath: string,
+		config: CustomResourceConfig,
+	][]
```

Comment:
I've said it before, but I really don't like us exposing these kinds of configs at this stage. We're trying to get a minimal working version out. Things like this should really be discussed in an issue first and put on the backlog.

Especially because it is a java-edition specific config that we're putting in the core config basically.

Reply:
As I said in #contributors, this field will be unsupported. 

As for the java-edition specific thing, I'll add the `pack` field here.

Reply:
> if it doesn't work it isn't blocking

We don't want to ship broken config code. If something isn't working, in my eyes it's blocking.

> As for the java-edition specific thing, I'll add the pack field here later today.

Not sure what you mean with this and how it addresses my concerns.

Reply:
I disagree. Here, all of this code is pretty flat and straight forward, I'll do some testing later today to make sure it doesn't blow up.

`pack: 'datapack'` would be the only value that would actually do anything for now, with `pack: 'java_resource_pack` being implemented eventually.

Reply:
oh right, this is already how its setup, it uses `pack`. no changes here are needed